### PR TITLE
chore: mypy fixes

### PR DESCRIPTION
**Pull request by kumaraditya303**

### CODE REVIEW

File: pytest_playwright/pytest_playwright.py [hunk 1]
```diff
@@ -18,6 +18,10 @@
 from typing import Any, Callable, Dict, Generator, List, Optional
 
 import pytest
+from pytest import Item
+from _pytest.config import Config
```
Comment (pavelfeldman) [inline pytest_playwright/pytest_playwright.py:22]: I don't see where pytest recommends using their private types. Is there a link?

Comment (kumaraditya303) [reply pytest_playwright/pytest_playwright.py:22]: It is used by the official pytest plugins as you can see here https://github.com/pytest-dev/pytest-print/blob/main/src/pytest_print/__init__.py

Comment (pavelfeldman) [reply pytest_playwright/pytest_playwright.py:22]: Yeah, I can see how pytest can use its own private apis. But we are not pytest, so we can't do that.

**Comment by pavelfeldman** [comment]
Closing since we should not use private pytest apis.

**Pull request closed without merging**